copier/main.py

```diff
@@ -144,6 +151,7 @@ class Worker:
     overwrite: bool = False
     pretend: bool = False
     quiet: bool = False
+    diff: bool = False
```

Comment:
This is a nice addition. I don't think it's needed to add a new flag for it.

Reply:
Just default to always diff when we can?